code/modules/client/preferences_savefile.dm

```diff
@@ -165,7 +171,7 @@
 	show_typing		= sanitize_integer(show_typing, FALSE, TRUE, initial(show_typing))
 	ghost_hud 		= sanitize_integer(ghost_hud, NONE, MAX_BITFLAG, initial(ghost_hud))
 	windowflashing	= sanitize_integer(windowflashing, FALSE, TRUE, initial(windowflashing))
-	key_bindings	= sanitize_islist(key_bindings, deepCopyList(GLOB.keybinding_list_by_key))
+	key_bindings	= sanitize_islist(key_bindings, null)
```

Comment:
This does not do any sanitization at all

Reply:
It checks its a list? If not returns null which is what i want (and what we did beforehand)

Reply:
it shouldn't return null, but an empty list instead